src/components/InvalidInputCard/index.jsx

```diff
@@ -0,0 +1,10 @@
+function InvalidInputCard() {
+    return (
+        <div>
```

Comment:
Same here, if you're not adding tailwind classes then wrapping it in a div will just create an invisible box around it